packages/esm-patient-forms-app/src/htmlformentry/html-form-entry-wrapper.component.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import styles from './html-form-entry-wrapper.scss';
+
+interface HtmlFormEntryWrapperProps {
+  closeWorkspace: () => void;
+  src: string;
+}
+
+const HtmlFormEntryWrapper: React.FC<HtmlFormEntryWrapperProps> = ({ closeWorkspace, src }) => {
+  useEffect(() => {
+    const iframe = document.querySelector('iframe');
+    iframe.addEventListener('load', () => {
```

Comment:
Rather than using `addEventListener`, etc. it's probably better to add the function to the `iframe`'s `onLoad` propery. That way the event handler is correctly cleaned up when the iframe is removed from the DOM.

```ts
onLoad={() => {
  const dashboard = iframe.current.contentDocument;
 // etc.
}}
```